test-requirements.txt

```diff
@@ -7,7 +7,7 @@ gen3rbac==0.1.2
 
 -e git+https://github.com/uc-cdis/cdisutils-test.git@fdc53fb3d5333d53f625f0f0712eaa2fa19c803f#egg=cdisutilstest
 -e git+https://github.com/uc-cdis/cdis-python-utils.git@0.1.7#egg=cdispyutils
--e git+https://github.com/uc-cdis/indexd.git@28fa621a924952be902628cfc5c1186bda38e922#egg=indexd
+-e git+https://github.com/uc-cdis/indexd.git@master#egg=indexd
```

Comment:
can you pin it pls

Reply:
the version in setup.py is not the same as in this file

Reply:
Oh, are versions of `indexclient` and `indexd` matched?